partiql-eval/src/plan.rs

```diff
@@ -92,19 +92,27 @@ impl EvaluatorPlanner {
                 as_key,
                 at_key.as_ref().unwrap(),
             )),
-            BindingsOp::Join(logical::Join { kind, on }) => {
+            BindingsOp::Join(logical::Join {
+                kind,
+                left,
+                right,
+                on,
+            }) => {
                 let kind = match kind {
-                    JoinKind::Inner => EvalJoinKind::Inner,
+                    JoinKind::Cross | JoinKind::Inner => EvalJoinKind::Inner,
                     JoinKind::Left => EvalJoinKind::Left,
                     JoinKind::Right => EvalJoinKind::Right,
                     JoinKind::Full => EvalJoinKind::Full,
-                    JoinKind::Cross => EvalJoinKind::Cross,
-                    JoinKind::CrossLateral => EvalJoinKind::CrossLateral,
```

Comment:
Model CROSS JOINs as INNER JOINs as mentioned by equivalence mentioned in [section 5.3 of spec](https://partiql.org/assets/PartiQL-Specification.pdf#subsection.5.3).

Reply:
Also as a comment in the code?

Reply:
Added comment to the code